applications/crossbar/doc/resources.md

```diff
@@ -80,6 +80,8 @@ Key | Description | Type | Default | Required | Support Level
 `require_flags` | When set to true this resource is ignored if the request does not specify outbound flags | `boolean()` |   | `false` |  
 `rules.[]` |   | `string()` |   | `false` |  
 `rules` | A list of regular expressions of which one must match for the rule to be eligible, they can optionally contain capture groups | `array(string())` | `[]` | `false` |  
+`rules_test.[]` |   | `string()` |   | `false` |  
+`rules_test` | A list of regular expressions of which if matched denotes a test rule | `array(string())` | `[]` | `false` |  
```

Comment:
Can you add a section below the schemas that talks about what the rules_test regexps are for? Something to explain the `why` one might use them?

Reply:
Updated multiple docs sections in latest commit.